plugins/MachineSettingsAction/MachineSettingsAction.qml

```diff
@@ -0,0 +1,475 @@
+// Copyright (c) 2016 Ultimaker B.V.
```

Comment:
Copyright here is different from the other bits. Is this copyrighted by UM or by you?